app/app.go

```diff
@@ -1,6 +1,12 @@
 package app
 
 import (
+	distr "github.com/cosmos/cosmos-sdk/x/distribution"
+	"github.com/cosmos/cosmos-sdk/x/evidence"
+	evidencekeeper "github.com/cosmos/cosmos-sdk/x/evidence/keeper"
+	evidencetypes "github.com/cosmos/cosmos-sdk/x/evidence/types"
+	"github.com/cosmos/cosmos-sdk/x/slashing"
+	slashingkeeper "github.com/cosmos/cosmos-sdk/x/slashing/keeper"
```

Comment:
I think it would be better to group the imports below where they got initially commented out to be more visual.

I suggest formatting all the imports in the codebase: https://github.com/zeta-chain/node/issues/851 so maybe all of this could be handled there.

Reply:
Makes sense , I like the idea of grouping , Might not be worth the time on this PR since we are not merging , but we do this in v7